lib/publiccloud/provider.pm

```diff
@@ -264,14 +264,16 @@ sub terraform_apply {
     my $image                = $self->get_image_id();
     my $ssh_private_key_file = '/root/.ssh/id_rsa';
     my $name                 = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $create_extra_disk    = get_var('PUBLIC_CLOUD_FIO') ? 'true' : 'false';
```

Comment:
If it is possible, I would like to avoid this implicit adding of parameters when we call `terraform_apply()`.

The idea behind this, if I create a different test which needs extra dist, but it's not FIO.
e.g.
```
$provider->create_instance(use_extra_disks => {size => 100 } );
```

Reply:
done